packages/mc-scripts/config/create-webpack-config-for-production.js

```diff
@@ -79,16 +79,56 @@ module.exports = ({
     optimization: {
       minimizer: [
         new TerserPlugin({
+          terserOptions: {
```

Comment:
Differences in sizes brought by those changes:

<img width="562" alt="Screenshot 2019-05-30 at 21 25 27" src="https://user-images.githubusercontent.com/1110551/58660549-fccf8d80-8325-11e9-9235-cd531cda72ee.png">